### ISSUE TITLE
`apt upgrade pyqt4` failed, but at least two others are ok(?)

### ISSUE THREAD

**Issue by maphew** [maphew/apt#34]
Curious, `apt upgrade pyqt4` failed, in the sense that it decided there was nothing to do, while `apt upgrade gpsbabel` and another package which I don't remember, worked just fine. Workaround to just remove the pyqt4 package and then install/upgrade it again was quick and easy, but still should have worked in the first place.

```
B:\code\apt>apt new

*** Warning: 'grass71-dev' is missing 'install' entry in setup.ini. This might cause problems.

Package             Installed   Available
-----------------   ---------   ----------
gdal                1.11.1-5    (1.11.2-1)
gdal-filegdb        1.11.1-5    (1.11.2-1)
gdal-python         1.11.1-5    (1.11.2-1)
gpsbabel            1.4.2-1     (1.4.4-1)
grass               6.4.4-1     (7.0.0-1)
osg-libs            3.0.0-1     (3.1.7-1)
osgearth-libs       2.0.0-1     (2.4-1)
pyqt4               4.8.3-2     (4.11.3-1)
python-numpy        2.7.1.5.1-1 (2.7.1.7.0-1)
python-qscintilla   2.6.2-2     (2.8.4-1)
qgis                2.6.0-1     (2.8.1-1)
qgis-common         2.4.0-1     (2.8.1-1)
qgis-dev            2.7.0-21    (2.9.0-34)
qgis-grass-plugin   2.4.0-1     (2.8.1-1)
qscintilla          2.6.2-2     (2.8.4-1)
qt4-libs            4.7.1-6     (4.8.6-1)

B:\code\apt>apt upgrade pyqt4

*** Warning: 'grass71-dev' is missing 'install' entry in setup.ini. This might cause problems.

+++ get_requires() is depcrecated. Please use get_all_dependencies().
PKGS: Checking install status: pyqt4
         pyqt4 - True
REQS: Checking dependencies installed: python-tools qt4-libs shell python-testsuite python setup zlib openssl python-cor
e python-help python-tcltk sqlite3 msvcrt
         python-tools - True
         qt4-libs - True
         shell - True
         python-testsuite - True
         python - True
         setup - True
         zlib - True
         openssl - True
         python-core - True
         python-help - True
         python-tcltk - True
         sqlite3 - True
         msvcrt - True

Packages and required dependencies are installed.

pyqt4               4.11.3-1

python-tools        2.7.4-1
qt4-libs            4.8.6-1
shell               1.0.0-13
python-testsuite    2.7.4-1
python              2.7.4-2
setup               1.0.7-10
zlib                1.2.3-1
openssl             1.0.1g-1
python-core         2.7.4-3
python-help         2.7.4-1
python-tcltk        2.7.4-1
sqlite3             3.7.17-1
msvcrt              1.0.1-12
```

**Comment by maphew** [comment]
possibly related: on another computer `apt upgrade all` only did a few of the packages. Needed to repeat several times before it got all of them. 

Something broken in what is returned from `get_new()`? or `install()` doesn't handle multiple incoming packages anymore?

This is the only issue remaining to fix before putting out a new release.

**Comment by amr66** [comment]
we have version check in get_new! it does not work!

**Comment by maphew** [comment]
should be fixed now with update to `install()`, bc20b72455b7bf1745d19026dbe59ae1f315b474. Closing because it now works for me,  please re-open if different for you.
